src/redux/actions/concepts/dictionaryConcepts.js

```diff
@@ -215,18 +215,16 @@ export const updateConcept = (conceptUrl, data, history) => async (dispatch) =>
   try {
```

Comment:
Is there anywhere you update `isFetching` to false when the call is successful?

Reply:
`isFetching` is an action creator that is dispatched with an argument of `true` when fetching and an argument of  `false` when response has been received. It is used for indicating API call status for things like AJAX loader icons.